### DISCUSSION TITLE
Nyugtamásolat nyomtatása

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#43] [Kérdések és válaszok]
Ha a vevő nem ad titkosító kulcsot, és nem kér másolatot, kötelező-e a nyugtamásolat nyomtatása? A nyugtaadási kötelezettségnek ekkor hogyan tehet eleget az üzemeltető?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
